amplipi/ctrl.py

```diff
@@ -229,6 +232,23 @@ def reinit(self, settings: models.AppSettings = models.AppSettings(), change_not
       logger.info("not lms mode")
       self.lms_mode = False
 
+    # determine if logs are persistant, based on the contents of a file
+    try:
+      conf_path = '/etc/systemd/journald.conf'
+      confparse = configparser.ConfigParser(strict=False, allow_no_value=True)
+      confparse.read(conf_path)
+
+      storage_value = confparse.get("Journal", "Storage")
+      if storage_value == "volatile":
+        self.persist_logs = False
+      elif storage_value == "persistent":
+        self.persist_logs = True
+      else:
+        self.toggle_persist_logs(False)
+    except Exception as exc:
+      logger.exception("Error reading persist_logs flag, setting to False (volatile logs). Exception: {exc}")
+      self.toggle_persist_logs(False)
```

Comment:
Attempts to read the config file to see what the current mode is. If it cannot be read, default to not persisting logs